rebar.config

```diff
@@ -22,7 +22,8 @@
     {httpc_aws, {git, "https://github.com/helium/httpc-aws.git", {branch, "master"}}},
     {websocket_client, "1.4.2"},
     {erlang_v8, {git, "https://github.com/Vagabond/erlang_v8.git", {branch, "master"}}},
-    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "master"}}},
+    {blockchain,
+        {git, "https://github.com/helium/blockchain-core.git", {branch, "mj/ul-direct-send-2"}}},
```

Comment:
do we want to change this back to master?

Reply:
good catch